Submitted By: Ryan Oliver <ryan.oliver@pha.com.au>
Date: 2005-05-03
Initial Package Version: 2.3.4
Origin: glibc cvs 
Upstream Status: See Origin
Description: 

(RO: Modified changelog entry)
2005-02-14  Alan Modra  <amodra@bigpond.net.au>

	* sysdeps/powerpc/powerpc32/dl-machine.h (elf_machine_rela): Define
	with auto inline, and attribute always_inline.
	(elf_machine_rela_relative, elf_machine_lazy_rel): Likewise.

===================================================================
RCS file: /cvs/glibc/libc/sysdeps/powerpc/powerpc32/dl-machine.h,v
retrieving revision 1.11
retrieving revision 1.13
diff -u -r1.11 -r1.13
--- libc/sysdeps/powerpc/powerpc32/dl-machine.h	2005/01/25 10:42:04	1.11
+++ libc/sysdeps/powerpc/powerpc32/dl-machine.h	2005/02/17 01:16:34	1.13
@@ -204,7 +204,7 @@
    LOADADDR is the load address of the object; INFO is an array indexed
    by DT_* of the .dynamic section info.  */
 
-inline void
+auto inline void __attribute__ ((always_inline))
 elf_machine_rela (struct link_map *map, const Elf32_Rela *reloc,
 		  const Elf32_Sym *sym, const struct r_found_version *version,
 		  void *const reloc_addr_arg)
@@ -298,7 +298,7 @@
     }
 }
 
-static inline void
+auto inline void __attribute__ ((always_inline))
 elf_machine_rela_relative (Elf32_Addr l_addr, const Elf32_Rela *reloc,
 			   void *const reloc_addr_arg)
 {
@@ -306,7 +306,7 @@
   *reloc_addr = l_addr + reloc->r_addend;
 }
 
-static inline void
+auto inline void __attribute__ ((always_inline))
 elf_machine_lazy_rel (struct link_map *map,
 		      Elf32_Addr l_addr, const Elf32_Rela *reloc)
 {
